papermill/cli.py

```diff
@@ -111,6 +111,11 @@ def print_papermill_version(ctx, param, value):
 @click.option(
     '--start_timeout', type=int, default=60, help="Time in seconds to wait for kernel to start."
 )
+@click.option(
+    '--execution_timeout',
```

Comment:
This should be `--execute-timeout` for the cli (click will convert it to `execution_timeout` automatically)

Reply:
Since no one was looking at PRs for a while over the holidays, I'll make this change now and get this merged now

Reply:
Thanks 👍